travis/script.sh

```diff
@@ -22,7 +22,7 @@ for f in ${CHANGED_FORMULAE};do
 #    echo "Clearing previously installed/cached formula ${f}..."
 #    brew uninstall --force --ignore-dependencies ${f} || true
 #  fi
-  if [[ grep -Fxq "$f" travis/skip-formulas.txt ]]; then
+  if grep -Fxq "$f" travis/skip-formulas.txt; then
```

Comment:
Would it make more sense to filter out skip formulas in the changed_formulas.sh file, instead of doing the skip check manually at each step?

That way we could always know that CHANGED_FORMULAE is the correct list of detected changes, manual rebuilds, and without skipped formulae?

Reply:
done